infrastructure/main.tf

```diff
@@ -40,6 +40,11 @@ locals {
   #endregion
 
   sku_size = "${var.env == "prod" || var.env == "sprod" || var.env == "aat" ? "I2" : "I1"}"
+  storage_account_name        = "${data.terraform_remote_state.shared_infra.storage_account_name}"
+  storage_account_primary_key = "${data.terraform_remote_state.shared_infra.storage_account_primary_key}"
+  storage_account_url_prod    = "https://bulkscan.platform.hmcts.net"
+  storage_account_url_notprod = "https://bulkscan.${var.env}.platform.hmcts.net"
```

Comment:
it would be nice to call this as storage_account_url_nonprod

Reply:
nonprod has connotations of being thought of as either aat or the nonprod subscription, this is for any environment that is not prod